extensions/blocks/opentable/edit.js

```diff
@@ -144,7 +144,6 @@ function OpenTableEdit( {
 				<div className={ `${ defaultClassName }-overlay` }></div>
 				<iframe
 					title={ sprintf( __( 'Open Table Preview %s', 'jetpack' ), clientId ) }
-					scrolling="no"
```

Comment:
I had a feeling this might crop up across multiple embeds and also on various browsers. We would probably have to explicitly set a `style` on each embed instead. It doesn't feel like much of a win at this point given that browsers will maintain backwards compatibility with this property for the foreseeable future.